src/app/(protected)/presentation/[presentationId]/_components/editor-sidebar/left-sidebar/LayoutPreview.tsx

```
@@ -0,0 +1,56 @@
+"use client"
+
+import { ScrollArea } from '@/components/ui/scroll-area'
+import { Skeleton } from '@/components/ui/skeleton'
+import { useSlideStore } from '@/store/useSlideStore'
+import React, { useEffect, useState } from 'react'
+
+type Props = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)